.github/ISSUE_TEMPLATE/2_anything-else.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Anything Else
+about: Submit an issue for a new feature or existing axis revision. 
+title: ''
+labels: '--review-axis'
```

Comment:
May be best not to auto label the anything else category as something that requires an axis review. There might be repo admin issues (CI tooling, release/distribution issues, etc) that fall in this category.

Reply:
Ok. We apply auto label in g/fonts repo as an introductory help, and then we review and change them. I choose this one as it was the second case in numbers in the current issues

Reply:
Up to you.  Maybe less work to remove the few that are incorrect than add many that are required.  Submitter can do it too.

Reply:
I think we could leave it like that and test it. We could modify this later if needed :)